compiler/optimizer/CopyPropagation.cpp

```diff
@@ -1117,6 +1117,19 @@ void TR_CopyPropagation::commonIndirectLoadsFromAutos()
 
    }
 
+
+void preserveKnownObjectInfo(TR::Compilation *comp, TR::Node *node, TR::SymbolReference *oldSymRef, TR::SymbolReference *newSymRef)
```

Comment:
Would you mind declaring this function `static`?

Reply:
done